plugins/modules/mysql_user.py

```diff
@@ -294,7 +294,15 @@
 # password=n<_665{vS43y
 '''
 
-RETURN = '''#'''
+RETURN = r'''
+queries:
+  description: List of executed queries which modified DB's state.
+  returned: if executed successfully
+  type: list
+  sample: ["CREATE USER 'db_user2'@'localhost' IDENTIFIED WITH mysql_native_password AS '********'"]
+  version_added: '1.1.0'
```

Comment:
```suggestion
  version_added: '1.3.0'
```
I released 1.2.0 two days ago, so it should be the next minor release